packages/notched-outline/index.tsx

```diff
@@ -21,46 +21,45 @@
 // THE SOFTWARE.
 import * as React from 'react';
 import * as classnames from 'classnames';
-// @ts-ignore no .d.ts file
-import {MDCNotchedOutlineFoundation} from '@material/notched-outline/dist/mdc.notchedOutline';
+import {MDCNotchedOutlineFoundation} from '@material/notched-outline/foundation';
+import {MDCNotchedOutlineAdapter} from '@material/notched-outline/adapter';
 
 export interface NotchedOutlineProps {
   className: string,
-  isRtl: boolean,
   notch: boolean,
-  notchWidth: number
+  notchWidth: number,
+  children?: React.ReactNode,
 };
 
 interface NotchedOutlineState {
   classList: Set<string>
+  actualWidth: number | null,
 };
 
 export default class NotchedOutline extends React.Component<
   NotchedOutlineProps,
   NotchedOutlineState
   > {
-  foundation_: MDCNotchedOutlineFoundation;
+  foundation_!: MDCNotchedOutlineFoundation;
   outlineElement_: React.RefObject<HTMLDivElement> = React.createRef();
```

Comment:
This doesn't need to be here anymore

Reply:
without it typescript throws an error

`Property 'foundation_' does not exist on type 'NotchedOutline'.`

Reply:
I am referring to `outlineElement_: React.RefObject<HTMLDivElement> = React.createRef();`